superset/charts/commands/exceptions.py

```diff
@@ -56,6 +56,23 @@ def __init__(self, human_readable: str) -> None:
         )
 
 
+class TimeDeltaUnclearError(ValidationError):
+    """
+    Time delta is in valid error.
```

Comment:
typo "invalid", better use "unclear" for consistency

Reply:
Nice catch, thank you.